docs/source/conf.py

```diff
@@ -2514,6 +2514,8 @@
     # torch.distributed.checkpoint.hf_storage
     "HuggingFaceStorageReader",
     "HuggingFaceStorageWriter",
+    # torch.distributed.checkpoint.quantized_hf_storage
+    "QuantizedHuggingFaceStorageReader",
```

Comment:
You should NOT add any entry to this file.
It means that things are not documented properly on the website.

Please add  a follow up PR to properly document these APIs.